grpc/service_config/service_config.proto

```diff
@@ -196,6 +196,21 @@ message PickFirstConfig {}
 // Configuration for round_robin LB policy.
 message RoundRobinConfig {}
 
+// Configuration for weighted_round_robin LB policy.
```

Comment:
For the same reason we prefixed it with "client_side" in Envoy, can we mention it's 'client-side' in the comment here?

Reply:
Not necessary in this case, because this is the only WRR policy that gRPC plans to support.